### DISCUSSION TITLE
Would be nice to have OWASP-related version

### DISCUSSION THREAD

**Discussion by wpawlikowski** [pwndoc/pwndoc#380] [Ideas]
Hi,

Been using pwndoc for a while, but always have to tune the system and add some code in order to work as I'd like to. Would be cool to have version / edition / configuration (whatever works) dedicated for appsec. Something like OWASP Risk Rating + OWASP Top 10 / ASVS. I can work on it during my free time dedicated for research, so if anyone wants to collaborate feel free to drop me a line

**Comment by peregrinus** [comment]
I just used custom finding fields with a select option and associated specific findings with an OWASP category. then had that table auto-generate:
![image](https://user-images.githubusercontent.com/1098005/211436801-f8afdf53-e261-4801-a9be-7db8e9b0fb77.png)

ASVS is another story. that would be a significant body of work to include all levels.

**Reply by HeadGapped** [reply]
Would you be able to show a screenshot of the template so I can get an understanding on how you are achieving this. Thank you.
